src/targetWidths.js

```diff
@@ -1,18 +1,3 @@
-function targetWidths() {
-  const resolutions = [];
-  let prev = 100;
-  const INCREMENT_PERCENTAGE = 8;
-  const MAX_SIZE = 8192;
+import ImgixClient from "@imgix/js-core";
 
-  const ensureEven = (n) => 2 * Math.round(n / 2);
-
-  while (prev <= MAX_SIZE) {
-    resolutions.push(ensureEven(prev));
-    prev *= 1 + (INCREMENT_PERCENTAGE / 100) * 2;
-  }
-
-  resolutions.push(MAX_SIZE);
-  return resolutions;
-}
-
-export default targetWidths();
+export default ImgixClient.targetWidths();
```

Comment:
nice!  🎉

Reply:
Yeah, I think we might change this setup (probably don't need a whole file to do this now).

Reply:
chefskiss.jpg